.circleci/config.yml

```diff
@@ -21,9 +21,26 @@ executors:
   deploy-executor:
     docker: # todo: create new image endpoint, add this in admin tab for faster starts
     - image: *ci_support_image
-    working_directory: ~/repo/pepper-apis  
+    working_directory: ~/repo/pepper-apis
 
 commands:
+  install-tcell:
+    steps:
+      - run:
+          name: Download tcell files from bucket
+          command: |
+            vault read -format=json  secret/pepper/prod/v1/conf | jq -r .data.gcp.serviceKey | gcloud auth activate-service-account --key-file=-
```

Comment:
We're reading the `prod` secrets in vault. Does that mean CircleCI have access to our prod secrets? That's a bit concerning...

Reply:
Is there a way to restrict a vault token to reading certain paths? Maybe we should put the service account in a separate path so we isolate build/deploy secrets from actual application secrets? Also, I understand that this is a bit tangential to tcell, so I'm okay pushing this up and discussing this point separately